examples/robolectricExample/build.gradle

```diff
@@ -0,0 +1,51 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'de.undercouch:gradle-download-task:2.0.0'
+    }
+}
+
+apply plugin: 'de.undercouch.download'
+apply plugin: 'android-sdk-manager'
+apply plugin: 'com.android.application'
+apply plugin: 'android-command'
+apply plugin: 'com.neenbedankt.android-apt'
+apply plugin: 'realm-android'
+apply plugin: 'realm-robolectric'
+
+def currentVersion = file("${rootDir}/../version.txt").text.trim()
```

Comment:
Without this, it will be encountered the following error.
```
* Where:
Build file '/Users/dalinaum/project/realm-java/examples/robolectricExample/build.gradle' line: 50

* What went wrong:
A problem occurred evaluating project ':robolectricExample'.
> Could not find property 'currentVersion' on org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler_Decorated@48adb967.
```